NEWS

```diff
@@ -1,7 +1,7 @@
 XLConnect News
 --------------
 
-1.0.6 ???
+1.0.6 2022-07-27
```

Comment:
I guess will need to be updated if other changes will make it into the release

Reply:
Indeed, waiting on the other PR also to then update the date to whenever we're ready to tag & release